tests/ui/conftest.py

```diff
@@ -15,7 +15,7 @@
 
 @pytest.fixture(scope='session')
 def base_url(base_url):
-    return base_url or os.getenv('PYTEST_BASE_URL')
+    return base_url
```

Comment:
No need to override this fixture if we're just returning the base URL unaltered. The idea here was to call the live_server fixture and return its URL if the base URL is not set.